scripts/version/build_no.sh

```diff
@@ -26,12 +26,14 @@
 # Fail on first error
 set -e
 
-GIT_ROOT=$(git rev-parse --show-toplevel)
 DEV_BUILD_NUMBER=9999
-BUILD_NUMBER_FILE="${GIT_ROOT}/BUILD_NUMBER"
+
+SCRIPTPATH="$(cd "$(dirname "$0")" ; pwd -P)"
```

Comment:
That's okay, but I kinda like your suggestion of:
```
GIT_ROOT=$(git rev-parse --show-toplevel || pwd)
```
It's a nice trick, but this works too.